packages/app_preferences/pubspec.yaml

```
@@ -0,0 +1,31 @@
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  flutter_lints: ^6.0.0
+  build_runner: ^2.4.15
+  freezed: ^3.0.6
+  json_serializable: ^6.9.5
+  riverpod_generator: ^2.6.5
+  riverpod_lint: ^2.6.5
+
+flutter:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)